src/mvMetricsWindow.cpp

```diff
@@ -174,16 +177,18 @@ void mvMetricsWindow::drawWidgets()
             ImGui::Spacing();
             ImGui::Text("ImGui State Inputs");
 
+            //TODO: Finish here with migration!
```

Comment:
Is this TODO still valid? Is there anything left to do?

Reply:
Nope, it was only regarding different `ImGuiKey`s